### DISCUSSION TITLE
How can I send automatic e-mails

### DISCUSSION THREAD

**Discussion by Hibrix-net** [LiveHelperChat/livehelperchat#1916] [Q&A]
Hi,

I'm trying to setup the chat notifications in order to automatically send an email based on the default template called "Chat was closed" found in the "E-mail templates" section.

So I edited the department notifications and checked "Inform then chat is closed by operator, only mail notification is send."
I also filled the field from below with my e-mail, etc. But after closing the chats (clicking the "X Close chat") I don't receive anything.

What I'm doing wrong? Is it necessary to setup the SMTP server for that?
I'm able to send the chat transcript without setting the SMTP server though.

BTW: Is it possible to automatically send chat transcripts to user and/or operator after the end of the chat?

Kind regards,

**Comment by remdex** [comment]
1. It all works here. https://github.com/LiveHelperChat/livehelperchat/blob/master/lhc_web/lib/core/lhchat/lhchat.php#L1478 you can debug from here what you missed.
2. SMTP is not required if test mail is working for you.
3. To send e-mail to operator use same chat was closed section and for user notification you can just listen `chat.close` event and setup bot to send e-mail... https://doc.livehelperchat.com/docs/development/webhooks
